signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottleByToken
+
+sf.org.apm.numSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+
+  Dimension(s): `orgId`, `tokenId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceivedByToken
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceived
+
+sf.org.apm.grossSpanBytesReceivedByToken:
+  brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceivedByToken
+
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceived
+
+sf.org.apm.numSpanBytesReceivedByToken:
+ brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceivedByToken
+
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to a SignalFx ingest endpoint for a specific access token.
+  description: 'The number of calls to a SignalFx ingest endpoint for a specific access token.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCallsByToken
+
+sf.org.apm.numAddSpansCalls:
+  brief: The number of calls to a SignalFx ingest endpoint.
+  description: 'The number of calls to a SignalFx ingest endpoint.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numAddSpansCalls
+
+sf.org.apm.invalidSpansReceived:
+  brief: 
```

Comment:
Does this metric actually exist? It seems to be the same as numSpansDroppedInvalid. @mdubbyap Can you check where this is coming from?

Reply:
@mdubbyap following up.